### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ghost**
Remove IntelliJ folder, remove mac spam, remove huge ass unneeded whiteline from Board, remove unneeded colorWilderness, reorganize imports, use AsnycPlayerChatEvent instead of
PlayerChatEvent, use Material instead of item IDs, use double for damage
instead of int, remove unneeded SuppressWarnings annotiation and use
Bukkit directly instead of Bukkit.getServer() when getting player.

### CODE REVIEW

File: src/main/java/com/massivecraft/factions/listeners/FactionsEntityListener.java [hunk 1]
```diff
@@ -180,9 +210,9 @@ public void onEntityExplode(EntityExplodeEvent event) {
                 targets.add(center.getRelative(1, 0, 0));
                 targets.add(center.getRelative(-1, 0, 0));
                 for (Block target : targets) {
-                    int id = target.getTypeId();
+                    Material type = target.getType();
                     // ignore air, bedrock, water, lava, obsidian, enchanting table, etc.... too bad we can't get a blast resistance value through Bukkit yet
-                    if (id != 0 && (id < 7 || id > 11) && id != 49 && id != 90 && id != 116 && id != 119 && id != 120 && id != 130)
+                    if (type != Material.AIR && type != Material.BEDROCK && type != Material.WATER && type != Material.STATIONARY_WATER && type != Material.LAVA && type != Material.STATIONARY_LAVA && type != Material.OBSIDIAN && type != Material.PORTAL && type != Material.ENCHANTMENT_TABLE && type != Material.ENDER_PORTAL && type != Material.ENDER_PORTAL_FRAME && type != Material.ENDER_CHEST)
```
Comment (Ribesg) [inline src/main/java/com/massivecraft/factions/listeners/FactionsEntityListener.java]: You should have some static EnumSet and just call a single .contains() here

File: src/main/java/com/massivecraft/factions/Board.java [hunk 2]
```diff
@@ -308,23 +307,4 @@ public static boolean load() {
...
-
-
-
-
-
-
-
-
-
+}
```
Comment (Ribesg) [inline src/main/java/com/massivecraft/factions/Board.java:311]: Woops, you should leave one linebreak here

Comment (ghost) [inline src/main/java/com/massivecraft/factions/Board.java:311]: Thanks

File: src/main/java/com/massivecraft/factions/listeners/FactionsEntityListener.java (hunk 1 above)
Comment (ghost) [inline src/main/java/com/massivecraft/factions/listeners/FactionsEntityListener.java]: Thanks

**Comment by gravitylow** [comment]
Have you tested these changes? The chat event transition to Async specifically sometimes causes issues.

**Comment by Ribesg** [comment]
Forcing the chat to be sync may be dangerous now, so if it cause problems, better fix the problems than keeping sync chat.

**Comment by ghost** [comment]
Yes I did test and the async chat events did not cause any issues.

File: src/main/java/com/massivecraft/factions/FPlayer.java [hunk 3]
```diff
@@ -1,5 +1,13 @@
 package com.massivecraft.factions;
 
+import java.util.HashSet;
+import java.util.Set;
+
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+
```
Comment (drtshock) [inline src/main/java/com/massivecraft/factions/FPlayer.java:10]: Why reorder my imports? My formatter does them alphabetically.

**Comment by drtshock** [comment]
How did you test it?

EDIT: it's accessing Bukkit API that isn't thread safe.

**Comment by drtshock** [comment]
I'm wary about pulling this because of the thread safety. I did remove the white space and unneeded files in my own commit. Thanks for pointing them out (:

**Pull request closed without merging**